lib/bolt/project_manager.rb

```diff
@@ -17,12 +45,13 @@ def initialize(config, outputter, pal)
     def create(path, name, modules)
       require 'bolt/module_installer'
 
-      project      = Pathname.new(File.expand_path(path))
-      old_config   = project + 'bolt.yaml'
-      config       = project + 'bolt-project.yaml'
-      puppetfile   = project + 'Puppetfile'
-      moduledir    = project + '.modules'
-      project_name = name || File.basename(project)
+      project       = Pathname.new(File.expand_path(path))
```

Comment:
This is why I hate aligned operators :P

Reply:
Different strokes :P I just prefer them since it makes things more legible.